wg-ecosystem/meeting-notes/2020-08-06.md

```diff
@@ -0,0 +1,48 @@
+# Ecosystem WG
+
+Thursday August 6, 2020 (10:00 AM - 11:00 AM PT)
+
+## Attendees
+
+- @erickzhao
+- @HashimotoYT
+- @kilian 
+- @malept
+- @MarshallOfSound
+
+## Agenda
+
+### Follow-up
+
+### New Items
+
+- @erickzhao: New permanent meeting time (we're not alternating anymore)
```

Comment:
Need to update also https://github.com/electron/governance/blob/master/wg-ecosystem/README.md#meeting-schedule

Reply:
@HashimotoYT We didn't change the time on the README after daylight savings time, so pushing the meeting time back an hour actually made the README time correct. 🤯 🤯 🤯 🤯 🤯

Reply:
nvm, DuckDuckGo shows me UTC+2 instead of UTC+3